pkg/kubelet/status/status_manager_test.go

```diff
@@ -90,8 +90,25 @@ func generateRandomMessage() string {
 }
 
 func getRandomPodStatus() v1.PodStatus {
```

Comment:
I think the `getRandomPodStatus` function name is fairly misleading here. It gave me the impression that the pod status could be anything, but that's not actually true. Only the `message` associated with the pod status is random.
In reality, the `status` of the pod is that all containers all terminated with an execute of 0. If you don't mind, could you update the function name to reflect that?

Reply:
@mattjmcnaughton 
I have renamed the func.
Please take a look.